utilities/virt.py

```diff
@@ -2757,3 +2760,48 @@ def wait_for_virt_handler_pods_network_updated(
         )
         raise
     return False
+
+
+def create_vm_with_nginx_service(chaos_namespace, admin_client, utility_pods, node, node_selector_label=None):
```

Comment:
1. please add typing (new functions under utilities need to include typing)
2. please rename `chaos_namespace` as this is a  generic function